src/main/java/org/jabref/gui/exporter/ExportCustomizationDialogViewModel.java

```diff
@@ -0,0 +1,98 @@
+package org.jabref.gui.exporter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+
+import org.jabref.gui.AbstractViewModel;
+import org.jabref.gui.DialogService;
+import org.jabref.logic.exporter.TemplateExporter;
+import org.jabref.logic.journals.JournalAbbreviationLoader;
+import org.jabref.preferences.PreferencesService;
+
+public class ExportCustomizationDialogViewModel extends AbstractViewModel {
+
+    //TODO: Use JavaFX SortedList instead of SimpleListPreperty for exporters.  There is no SimpleSortedListProperty.
+    private final ListProperty<ExporterViewModel> exporters = new SimpleListProperty<>(FXCollections.observableArrayList());
+    private final ListProperty<ExporterViewModel> selectedExporters = new SimpleListProperty<>(FXCollections.observableArrayList());
+
+
+    private final PreferencesService preferences;
+    private final DialogService dialogService;
+    private final JournalAbbreviationLoader loader;
+
+    public ExportCustomizationDialogViewModel(PreferencesService preferences, DialogService dialogService, JournalAbbreviationLoader loader) {
+        this.preferences = preferences;
+        this.dialogService = dialogService;
+        this.loader = loader;
+        init();
+    }
+
+    public void loadExporters() {
+        List<TemplateExporter> exportersLogic = preferences.getCustomExportFormats(loader); //Var exportersLogic may need more descriptive name
+        for (TemplateExporter exporter : exportersLogic) {
+            exporters.add(new ExporterViewModel(exporter));
+        }
+    }
+
+    public void addExporter() {
+        Optional<ExporterViewModel> blankExporter = Optional.empty();
+        CreateModifyExporterDialogView dialog = new CreateModifyExporterDialogView(blankExporter, dialogService, preferences,
+                                                                                   loader);
+        Optional<Optional<ExporterViewModel>> exporter = dialogService.showCustomDialogAndWait(dialog);
+        if (exporter.isPresent() && exporter.get().isPresent()) {
+            exporters.add(exporter.get().get());
+        }
+    }
+
+    /**
+     * Open dialog to modify exporter, which is the same as adding ane exporter but passes in
+     * a non-empty Optional of ExporterViewModel and sets the result into exporters.
+     */
+
+    public void modifyExporter() {
+        CreateModifyExporterDialogView dialog;
+        ExporterViewModel exporterToModify = selectedExporters.get(0);
```

Comment:
This throws an exception when no exporter is selected. I start the method with `if (selectedExporters.isEmpty()) {return;}` because there is nothing that we can do if the users does not tell us what exporter to edit.